packages/story-editor/src/components/canvas/test/singleSelectionMoveable.js

```diff
@@ -17,7 +17,8 @@
  * External dependencies
  */
 import { render, act, waitFor } from '@testing-library/react';
-import { MockMoveable, withOverlay } from '@googleforcreators/moveable';
+import { withOverlay } from '@googleforcreators/moveable';
+import { MockMoveable } from 'react-moveable'; // eslint-disable-line import/named
```

Comment:
For context: we're working on adding explanations everywhere when disabling ESLint rules. See #10485 

```suggestion
import { MockMoveable } from 'react-moveable'; // eslint-disable-line import/named -- This is a custom Jest mock
```